.github/workflows/integrationTest.yml

```diff
@@ -370,42 +367,53 @@ jobs:
       - name: Verify Terraform version
         run: terraform --version
 
-      - name: Terraform init
-        run: terraform init
+      - name: Set up terraform
+        if: steps.ec2-linux-integration-test.outputs.cache-hit != 'true'
+        uses: hashicorp/setup-terraform@v1
```

Comment:
Since we don't use the global working directory anymore, does this work? Will it be able to set up from the root directory?

Reply:
This works but it can be deleted since ubuntu comes with terraform installed. What this action does is install and configure terraform but by default it is already done. This would only help if we need to set the terraform version manually.

Reply:
I will delete.